### DISCUSSION TITLE
Podman 兼容/支持问题？

### DISCUSSION THREAD

**Discussion by lyimoexiao** [1Panel-dev/1Panel#4528] [Ideas]
# 为什么

作为兴起开源之秀，Podman对比Docker更为开放，且在某些方面是要优于Docker的（例如安全、性能等...），再加上Podman定位也是与Docker兼容，因此在使用上面尽量靠近Docker。所以即便使用了Podman，也可以做到几乎与Docker一样的兼容性（？）

且Podman的开源协议相对应Docker是更加友好的，也十分符合1Panel这种开源项目的使用。

所以个人方面认为Podman为未来所趋势。

# 怎么做

当然我知道目前1Panel直接从Docker迁移至Podman并完成兼容不是“改几个东西”那么简单，所以是否能考虑在当前的开发进展下新开一个分支用于对Podman的兼容适配进行开发测试验证（可以发起个投票征求下各位的意见）？待时机成熟再并入主线进行开发，并后续给用户提供兼容选项

最后非常感谢 1Panel 团队的辛苦开发！

**Comment by wanghe-fit2cloud** [comment]
感谢反馈，我们会在 v2.0 大版本重构时考虑上述需求的。

**Reply by lyimoexiao** [reply]
希望能够早日支持上~
非常感谢你们的辛勤开发

**Comment by wanghe-fit2cloud** [comment]
https://github.com/1Panel-dev/1Panel/issues/1070
